app/validation/validator.py

```diff
@@ -48,9 +47,9 @@ def _validate_json_against_schema(self, json_to_validate):
             validate(json_to_validate, self.schema, resolver=resolver)
             return []
         except ValidationError as e:
-            return ['Schema Validation Error! does not validate against schema. Error [{}]'.format(e)]
+            return ['Message [{}] Path [{}]'.format(e.message, e.path)]
         except SchemaError as e:
-            return ['JSON Parse Error! Error [{}]'.format(e)]
+            return ['{}'.format(e)]
```

Comment:
If we're only ever returning one error we probably don't need to be wrapping it in an array here